igor-web/src/main/groovy/com/netflix/spinnaker/igor/jenkins/JenkinsBuildMonitor.groovy

```diff
@@ -198,8 +198,7 @@ class JenkinsBuildMonitor implements PollingMonitor {
                                                         Project oldProject = new Project(name: project.name, lastBuild: build)
                                                         if (build.number != lastBuild
                                                                 || (build.number == lastBuild && cachedBuild.lastBuildBuilding != build.building)) {
-                                                            echoService.postEvent(
-                                                                    new BuildEvent(content: new BuildContent(project: oldProject, master: master)))
+                                                            postEvent(echoService, cachedBuild, oldProject, master)
```

Comment:
I'm not sure this handles new jobs correctly. If I have a new jenkins job with completed runs when the index runs, say build 1 and build 2, it would look like the events for those builds will get swallowed and not triggered again. Seeing that adding new jobs is more likely than losing redises, I'm not sure this is the best way to do this. Can we not just check the number of indexed builds per master and only do this when that particular master has 0 indexed builds?